sql/src/main/java/io/crate/expression/scalar/ArrayCatFunction.java

```diff
@@ -53,7 +54,17 @@ public static FunctionInfo createInfo(List<DataType> types) {
     }
 
     public static void register(ScalarFunctionModule module) {
-        module.register(NAME, new Resolver());
+        module.register(
+            Signature.builder()
```

Comment:
Do you think it would make sense to have a shortcut for the common case?

Like `Signature.scalar(NAME, DataType<?> ... types)` where the last type is the return type.

Reply:
Yes sure, I've planned to add them later on when moving more functions to the new registry.